resources/eventing/charts/event-publisher-proxy/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "event-publisher-proxy.fullname" . }}
+  labels: {{- include "event-publisher-proxy.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels: {{- include "event-publisher-proxy.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels: {{- include "event-publisher-proxy.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ include "event-publisher-proxy.serviceAccountName" . }}
+      containers:
+        - env:
+            - name: CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "event-publisher-proxy.secretName" . }}
+                  key: client-id
+            - name: CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "event-publisher-proxy.secretName" . }}
+                  key: client-secret
+            - name: TOKEN_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "event-publisher-proxy.secretName" . }}
+                  key: token-endpoint
+            - name: EMS_PUBLISH_URL
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "event-publisher-proxy.secretName" . }}
+                  key: ems-publish-url
+          image: "{{ .Values.global.image.repository }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+          name: event-publisher-proxy
+          ports:
+            - containerPort: 8080
```

Comment:
No dedicated port for metrics?

Reply:
So far no metrics ;)

Reply:
Metrics will come as part of this [issue](https://github.com/kyma-project/kyma/issues/9523)